scheduling_with_python.md

```diff
@@ -0,0 +1,49 @@
+# Scheduling using Python's schedule library
+
+Users may want to have the scripts in this repository run at certain frequencies. For example, a user may want to 
+run `delete_assignments.py` once a day to delete all completed assignments, or run `reset_stale_workers.py`. once a
+week to reset workers that aren't working to the correct status.
+
+On Windows, we recommend using Windows Task Scheduler for this action. See our blog post here [here](https://community.esri.com/groups/workforce-for-arcgis/blog/2020/05/14/schedule-tasks-for-workforce)
+for more on how to set that up.
+
```

Comment:
I think you should include how to use crontab here for Linux/Mac users. That seems like the industry standard. Like the add steps like [this article](https://www.jessicayung.com/automate-running-a-script-using-crontab/) for the workforce scripts.